src/Stoa.ts

```diff
@@ -553,22 +571,29 @@ class Stoa extends WebService {
      */
     private getTransactionFees(req: express.Request, res: express.Response) {
         const size: string = req.params.tx_size.toString();
-
+        let currency: any = String(req.query.currency);
+        if (req.query.currency === undefined) {
+            currency = CurrencyType.USD;
+        }
```

Comment:
```
let obj = { value: undefined };
let value: string = String(obj.value);
console.log(value);
```
The result is "undefined"

```suggestion
        let currency: string;
        if (req.query.currency === undefined) {
            currency = CurrencyType.USD;
        } else {
            currency = String(req.query.currency);
        }
```